src/main/java/Duke.java

```diff
@@ -1,10 +1,31 @@
+import duke.util.List;
+import duke.util.Parser;
+import duke.util.Ui;
+
 public class Duke {
```

Comment:
I believe we need to write javadocs for all public classes? This applies for all your other public classes!:smile: